recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# {% set version = "1.0.2.dev0" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+#  fn: obspy-{{ version }}.zip
+#  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  fn: condaforge_win.zip
+  url: https://github.com/megies/obspy/archive/condaforge_win.zip
```

Comment:
This is quite unusual. I would prefer if you use a canonical source here instead of a special source. If there are changes that are specific to package for conda-forge we can patch the source.